package.json

```diff
@@ -66,16 +68,15 @@
     "husky": "^0.14.3",
     "js-polyfills": "^0.1.33",
     "jsdoc-to-markdown": "^3.0.0",
-    "karma": "^1.3.0",
-    "karma-chrome-launcher": "^2.2.0",
+    "karma": "^3.0.0",
     "karma-env-preprocessor": "^0.1.1",
     "lint-staged": "^5.0.0",
-    "mocha": "^3.0.0",
-    "mochainon": "^1.0.0",
+    "mocha": "^5.2.0",
+    "mochainon": "^2.0.0",
     "prettier": "^1.8.2",
     "readable-stream": "^2.3.6",
     "require-npm4-to-publish": "^1.0.0",
-    "resin-config-karma": "^1.0.4",
+    "resin-config-karma": "^2.2.0",
```

Comment:
Not strictly necessary, but this would be a nice moment to switch it to [balena-config-karma](npmjs.com/package/balena-config-karma) (it's the same source as 2.3.1, just republished)

Reply:
Will do :+1: